app/models/concerns/current_tracked_by_changeset.rb

```diff
@@ -94,11 +106,17 @@ def current_tracked_by_changeset(kind_of_model_tracked: nil, virtual_attributes:
       else
         raise ArgumentError.new("must specify whether it's an entity or a relationship being tracked")
       end
-
       @virtual_attributes = virtual_attributes
     end
   end
 
+  def as_change
+    Hash[
+      slice(*self.class.changeable_attributes).
+      map { |k, v| [k.to_s.camelize(:lower).to_sym,v] }
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Space missing after comma.